src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -87,6 +87,79 @@
     }
     
     /**
+     * Generates the summary of the feedback sessions (in which at least one
+     * email has been sent to students) email for the given {@code courseId} for {@code student}
```

Comment:
Some changes possible:
- Generates the email containing the summary of ...
- Remove ~~(in which at least ... students)~~ because this is the call-site condition, and the method should be independent of it.
- Add `@param` documentation for both the parameters.